test/Microsoft.Framework.Logging.Test/LoggerFactoryExtensionsTest.cs

```diff
@@ -11,7 +11,7 @@ namespace Microsoft.Framework.Logging.Test
     public class LoggerFactoryExtensionsTest
     {
 #if ASPNET50
```

Comment:
Oh interesting. Yeah no point having the `#ifdef` in that case, but instead of that what would it take to have some coverage on `aspnetcore50`? I am especially interested in this case because some Reflection stuff could certainly be a bit different between the two. I'm not suggesting porting *all* the tests to that, but at least these new ones. If it's somehow super costly then we can reconsider.